makefiles/ktool.mk

```diff
@@ -0,0 +1,47 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS    += ktool
+# Get rid of this variable at some point
+KTOOL_TVERSION := 1.0.0rc0
+KTOOL_VERSION  := 1.0.0~rc0
+DEB_KTOOL_V    ?= $(KTOOL_VERSION)
+
+ktool-setup: setup
+	$(call GITHUB_ARCHIVE,cxnder,ktool,$(KTOOL_VERSION),$(KTOOL_TVERSION))
+	$(call EXTRACT_TAR,ktool-$(KTOOL_VERSION).tar.gz,ktool-$(KTOOL_TVERSION),ktool)
+
+ifneq ($(wildcard $(BUILD_WORK)/ktool/.build_complete),)
+ktool:
+	@echo "Using previously built ktool."
+else
+ktool: ktool-setup kimg4 pyaes pygments
```

Comment:
Add `python3`

Reply:
No. Dependencies already depend on python3, so invoking one will install the other

Reply:
So? You could in theory use `setup_base.sh` and have the depends built without python3.

Reply:
Not what I mean; this also happens with control files, so there's no need to specify the python3 depedency everywhere (?)

Reply:
That is not the point. Why specify `libintl8` dependency if it's gonna be installed by something else? Because it is a dependency.